docs/ml/upload-model.md

```diff
@@ -14,12 +14,24 @@ description: "Upload a Machine Learning model to Viam to use it with the ML Mode
 The ML model service works with models trained inside and outside the Viam app.
 To use a model that you have trained yourself outside the Viam app, [upload it as a new model privately or share it in the Viam registry](#upload-a-new-model-or-new-version).
 If you need to update a previously uploaded model, you can also [upload a new version](#upload-a-new-model-or-new-version).
-To work with the `tflite_cpu` ML model service, an ML model is comprised of a <file>.tflite</file> model file which defines the model, and optionally a <file>.txt</file> labels file which provides the text labels for your model.
```

Comment:
Should we add this info to the table or somewhere add info for each of the frameworks?

Reply:
Adding that + all the info steven said was needed to the upload model page in this pr https://github.com/viamrobotics/docs/pull/2628/files